src/components/screens/Service.tsx

```diff
@@ -32,7 +32,7 @@ const umlautSwitcher = (text: string) => {
 
   const replacedText = text.replace(UMLAUT_REGEX, (match: string) => umlautReplacements[match]);
 
-  return replacedText;
+  return replacedText?.replace('​', '');
```

Comment:
please add `?.replace('​', '')` directly to the line before when creating `replacedText`. thi reads much more accurate i guess.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/b224795fb9c4703537927397763d18a133ea3a6d